static/styles/subscriptions.scss

```diff
@@ -611,6 +611,27 @@ form#add_new_subscription {
             pointer-events: none;
             visibility: hidden;
         }
+
+        #sub_unsub_status {
+            display: inline-block !important;
+            height: auto !important;
+            width: auto !important;
+
+            .alert-error {
```

Comment:
Under what circumstances is this displayed?  And have you tested these colors in both night and day themes?

Reply:
Removed this redundant code. Will add it again, when working on error handling.